cms/static/js/factories/manage_users_lib.js

```diff
@@ -0,0 +1,159 @@
+        var unknownErrorMessage = gettext('Unknown'),
+            $createUserForm = $('#create-user-form'),
+            $createUserFormWrapper = $createUserForm.closest('.wrapper-create-user'),
+            $cancelButton;
+
+        // Our helper method that calls the RESTful API to add/remove/change user roles:
+        var changeRole = function(email, newRole, opts) {
+            var url = tplUserURL.replace('@@EMAIL@@', email);
+            var errMessage = opts.errMessage || gettext("There was an error changing the user's role");
+            var onSuccess = opts.onSuccess || function(data){ ViewUtils.reload(); };
+            var onError = opts.onError || function(){};
+            $.ajax({
+                url: url,
+                type: newRole ? 'POST' : 'DELETE',
+                dataType: 'json',
+                contentType: 'application/json',
+                notifyOnError: false,
+                data: JSON.stringify({role: newRole}),
+                success: onSuccess,
+                error: function(jqXHR, textStatus, errorThrown) {
+                    var message, prompt;
+                    try {
+                        message = JSON.parse(jqXHR.responseText).error || unknownErrorMessage;
+                    } catch (e) {
+                        message = unknownErrorMessage;
+                    }
+                    prompt = new PromptView.Error({
+                        title: errMessage,
+                        message: message,
+                        actions: {
+                            primary: { text: gettext('OK'), click: function(view) { view.hide(); onError(); } }
+                        }
+                    });
+                    prompt.show();
+                }
+            });
+        };
+
+        $createUserForm.bind('submit', function(event) {
+            event.preventDefault();
+            var email = $('#user-email-input').val().trim();
+            var msg;
+
+            if(!email) {
+                msg = new PromptView.Error({
+                    title: gettext('A valid email address is required'),
+                    message: gettext('You must enter a valid email address in order to add an instructor'),
+                    actions: {
+                        primary: {
+                            text: gettext('Return and add email address'),
+                            click: function(view) { view.hide(); $('#user-email-input').focus(); }
+                        }
+                    }
+                });
+                msg.show();
+                return;
+            }
+
+            if(_.contains(allUserEmails, email)) {
+                msg = new PromptView.Warning({
+                    title: gettext('Already a library team member'),
+                    message: _.template(
+                        gettext("{email} is already on the “{course}” team. If you're trying to add a new member, please double-check the email address you provided."), {
```

Comment:
Is "Recheck the email address if you want to add a new member" OK?

FYI I'm going to change this in the course user settings context too, as this was an existing string that I copied from there.